modules/core/osgi-service-tracker-collections/src/main/java/com/liferay/osgi/service/tracker/collections/internal/list/ServiceTrackerListImpl.java

```diff
@@ -174,8 +174,6 @@ public void removedService(
 			_serviceTrackerCustomizer.removedService(serviceReference, service);
 
 			_update(serviceReference, service, true);
-
```

Comment:
@rafaprax Please fix this in a separate ticket and add test to cover it.

Thanks.